### ISSUE TITLE
Will this signal ever be deallocated?

### ISSUE THREAD

**Issue by JRHeaton** [ReactiveCocoa/ReactiveCocoa#1394]
So I have a `RACCommand` for my button, which does some asynchronous stuff, and I'd like to enable this loading animation behavior for it while it's doing so. I currently am doing this:

``` objc
[self.userButton.rac_command.executing
    subscribeNext:^(NSNumber *executing) {
        if (executing.boolValue) {
            [self.userButton startLoading];
        } else {
            [self.userButton stopLoading];
        }
    }];
```

There are 2 questions I have about this:

1) Is there any way to do this without explicitly subscribing? I tried "lifting" this selector in response to the signal but it wants to send it as an argument, which obviously wouldn't work in this case.

2) Will this signal ever be deallocated? I don't think that executing ever sends complete, and even if the command itself tore down the signal when IT was deallocated, this issue still applies to more than just this scenario.

TL;DR: Must you must use `takeUntil:` or manually dispose of subscriptions for signals that are of "indefinite" nature, so to speak? For another concrete example, what if I used `rac_signalForSelector:`?
